torch/onnx/_internal/fx/diagnostics.py

```diff
@@ -21,6 +21,9 @@
 # Tensor(i64[s0, 64, (s1//2) - 2, (s1//2) - 2]) where s0 and s1 are symbolic
 # so we need to relax the length limit.
 _LENGTH_LIMIT: int = 120
+# NOTE: The following limits are for the number of items in a list, tuple or dict.
+# The limits are set to be small to avoid heavy workload.
+_CONTAINER_ITEM_LIMIT: int = 10
```

Comment:
I think we should reason that args/kwargs usually smaller than 10.

Reply:
rephrased